frontend/packages/dev-console/src/components/topology/components/groups/HelmReleaseGroup.tsx

```diff
@@ -12,61 +12,68 @@ import {
 
-const HelmReleaseGroup: React.FC<HelmReleaseGroupProps> = ({
-  element,
-  dragging,
-  filters,
-  onSelect,
-  selected,
-}) => {
+const HelmReleaseGroup: React.FC<HelmReleaseGroupProps> = ({ element, onSelect, selected }) => {
   const [hover, hoverRef] = useHover();
+  const [labelHover, labelHoverRef] = useHover();
   const { x, y, width, height } = element.getBounds();
-  const dragNodeRef = useDragNode()[1];
-  const dragLabelRef = useDragNode()[1];
-  const refs = useCombineRefs(dragNodeRef, hoverRef);
-  const filtered = useFilter(filters, { metadata: { name: element.getLabel() } });
-
-  const rectClasses = classNames('odc-helm-release', {
-    'is-selected': selected,
-    'is-hover': hover,
-    'is-filtered': filtered,
+  const [{ dragging }, dragNodeRef] = useDragNode({
+    collect: (monitor) => ({
+      dragging: monitor.isDragging(),
+    }),
   });
-
+  const [{ labelDragging }, dragLabelRef] = useDragNode({
+    collect: (monitor) => ({
+      labelDragging: monitor.isDragging(),
+    }),
+  });
+  const refs = useCombineRefs(dragNodeRef, hoverRef);
+  const [filtered] = useSearchFilter(element.getLabel());
   return (
-    <g>
+    <>
       <NodeShadows />
       <Layer id={dragging ? undefined : 'groups'}>
-        <rect
-          ref={refs}
-          className={rectClasses}
-          onClick={onSelect}
-          x={x}
-          y={y}
-          width={width}
-          height={height}
-          rx="5"
-          ry="5"
-          filter={createSvgIdUrl(
-            hover || dragging ? NODE_SHADOW_FILTER_ID_HOVER : NODE_SHADOW_FILTER_ID,
-          )}
-        />
+        <g
+          className={classNames('odc-helm-release', {
+            'is-dragging': dragging || labelDragging,
+            'is-selected': selected,
+            'is-filtered': filtered,
+          })}
+        >
+          <rect
+            ref={refs}
+            className="odc-helm-release__bg"
+            onClick={onSelect}
+            x={x}
+            y={y}
+            width={width}
+            height={height}
+            rx="5"
+            ry="5"
+            filter={createSvgIdUrl(
+              hover || labelHover || dragging || labelDragging
+                ? NODE_SHADOW_FILTER_ID_HOVER
+                : NODE_SHADOW_FILTER_ID,
+            )}
+          />
+        </g>
       </Layer>
       {element.getLabel() && (
-        <g onClick={onSelect}>
+        <g
+          ref={labelHoverRef}
+          onClick={onSelect}
+          className={classNames('odc-helm-release', {
+            'is-dragging': dragging || labelDragging,
+            'is-filtered': filtered,
+          })}
+        >
           <SvgBoxedText
-            className={classNames('odc-base-node__label', 'odc-helm-release__label', {
-              'is-filtered': filtered,
-              'is-dragging': dragging,
-            })}
+            className="odc-base-node__label"
             x={x + width / 2}
             y={y + height + 20}
```

Comment:
Other groupings use 30 here. Maybe a constant or default value in `SvgBoxedText`? (paddingX, paddingY, and truncate as well?)

Reply:
Let's improve this as part of a PR to to fix the label padding.